bindgen/codegen/helpers.rs

```diff
@@ -301,32 +301,27 @@ pub(crate) mod ast_ty {
         }
     }
 
-    pub(crate) fn float_expr(
-        ctx: &BindgenContext,
-        f: f64,
-    ) -> Result<TokenStream, ()> {
+    pub(crate) fn float_expr(f: f64) -> Result<TokenStream, ()> {
```

Comment:
According to this https://rust-lang.github.io/rust-clippy/master/index.html#/legacy_numeric_constants
This fix should also be applied for `f32` et al